build_process/node.md

```diff
@@ -1,19 +1,59 @@
 # Node
 
-## NPM Scripts
+## Managing Your Local Node Versions
 
-Our projects use different build systems (Gulp, Grunt, custom, etc), but many 
-times they have tasks that do similar things on every project. As developers 
-move from project to project we can save some cognitive overhead by 
-standardizing the commands we use for these common tasks. With node we can use 
-[npm scripts](https://docs.npmjs.com/misc/scripts) as shortcuts to our various 
+To manage your own Node versions we have a few recommendations, but you should do what works for you. Whichever method is reliable and comfortable is the right one.
+
+### Happy Path
+
+Use [`nvm`](https://github.com/nvm-sh/nvm). See documentation for [installation](https://github.com/nvm-sh/nvm#installation-and-update) and [usage](https://github.com/nvm-sh/nvm#usage). 
+
+### Secondary Options
+
+- [`n`](https://www.npmjs.com/package/n) - this is an `npm` install-able node-based tool. It's real nice.
+- [`brew`](https://brew.sh/) - this is a perfectly fine option. See [instructions](./managing-node-with-brew.md) for using `brew`.
```

Comment:
We should remove this from the laptop script soon.

Reply:
And come up with a migration plan/rollout strategy from brew -> npm.

Reply:
I will look into this.